scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -283,8 +283,20 @@ class ScanCode(name: String, config: ScannerConfiguration) : LocalScanner(name,
         }
     }
 
+    private fun getFileCount(result: JsonNode): Int {
+        // Handling for ScanCode 2.9.8+
+        result.get("headers")?.forEach { header ->
```

Comment:
Nit: Please replace `get()` with the indexing operator for consistency with existing code.